xbmc/addons/kodi-addon-dev-kit/include/kodi/libXBMC_addon.h

```diff
@@ -156,6 +156,7 @@ typedef struct CB_AddOn
   bool (*FileExists)(const void* addonData, const char *strFileName, bool bUseCache);
   int (*StatFile)(const void* addonData, const char *strFileName, struct __stat64* buffer);
   char *(*GetFileProperty)(const void* addonData, void* file, XFILE::FileProperty type, const char *name);
+  char **(*GetFilePropertyValues)(const void* addonData, void* file, XFILE::FileProperty type, const char *name, int *numPorperties);
```

Comment:
Sorry, to mention this, but ofc this change (inserting a new function in the middle of the function table) makes each and every binary addon that still uses old-style API (like PVR addons) incompatible and therefore an API bump is required.